index.html

```diff
@@ -14,46 +14,258 @@
+          <img src="img/Ellipse 12.png" alt="Eclipse-image" />
+        </div>
+
+        <div class="eclipse-13">
+          <img src="img/Ellipse 13.png" alt="Eclipse-image" />
+        </div>
+
+        <ul class="social-icons">
+          <li><span class="iconify" data-icon="iconoir:github"></span></li>
+          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
+          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
+          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
+          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+        </ul>
+
+        <div class="eclipse-4">
+          <img src="img/Ellipse 4.png" alt="Eclipse-image" />
+          <img src="img/Rectangle 5.png" alt="Eclipse-image" />
+        </div>
+      </section>
+    </section>
+
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <h2 class="canopy">CANOPY</h2>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
         </p>
-      </div>
-
-      <p class="connect">LET’S CONNECT</p>
-
-      <div class="social-icons">
-        <span class="iconify" data-icon="iconoir:github"></span>
-        <span class="iconify" data-icon="eva:linkedin-fill"></span>
-        <span class="iconify" data-icon="fontisto:angelist"></span>
-        <span class="iconify" data-icon="ei:sc-twitter"></span>
-        <span class="iconify" data-icon="bx:bxl-medium"></span>
-      </div>
-    </main>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javaScript</button></li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+
+      <!-- multipurpose section1 -->
+      <section class="multipurpose-section">
+        <div class="multipurpose-image"></div>
+
+        <h2 class="multi-Post">
+          Multi-Post<br />
+          Stories
+        </h2>
+        <div class="canopy-container">
+          <h2 class="canopy">CANOPY</h2>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javaScript</button></li>
+        </ul>
```

Comment:
Same issue as above
- [ ] Please fix the styling of these items, furthermore, they are not buttons they are labels or badges. 
- [ ] Please use the same class for all of theme, something like `tech-label` should be fine but thats up to you.
## Tech button fix
![label-fix](https://user-images.githubusercontent.com/28903404/143720740-37fb3da0-cb79-4d6d-83ed-a25006d4968e.gif)

As you can see from the little video, they need `padding`, ` background-color`, `border-radiious` and you need to remove the hardcoded sizes. The size will come from the text inside the div.